src/zcl/definition/cluster.ts

```diff
@@ -4120,6 +4120,7 @@ const Cluster: {
                     {name: 'datatype', type: DataType.uint8},
                     {name: 'fn', type: DataType.uint8},
                     {name: 'data', type: DataType.octetStr},
+                    {name: 'additionalValues', type: BuffaloZclDataType.LIST_TUYA_DATAPOINT_VALUES},
```

Comment:
In order not to update all of the existing converters, I chose to store additional DP values in an extra field.

Alternatively (and cleaner) one could just do:

```typescript
            dataResponse: {
                ID: 1,
                parameters: [
                    {name: 'seq', type: DataType.uint16},
                    {name: 'dpValues', type: BuffaloZclDataType.LIST_TUYA_DATAPOINT_VALUES},
                ],
            },
```

But this would break all existing TuYa converters (and, if I adapt these, I don't have a way to test them).